proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -537,16 +547,15 @@ def recovery_mass_phase_comp_bounds(b, t, p, j):
             self.flowsheet().config.time,
             doc="Material properties of feed-side interface at outlet",
             default=tmp_dict)
-        # Permeate properties
-        self.properties_permeate = self.config.property_package.state_block_class(
+        self.permeate_side.properties_permeate = self.config.property_package.state_block_class(
             self.flowsheet().config.time,
             doc="Material properties of permeate",
```

Comment:
Is this the state block for the mixed (average) permeate?
```suggestion
            doc="Material properties of mixed permeate",
```

Reply:
I included your suggested change and renamed `permeate_side.properties_permeate` to `permeate_side.properties_mixed`, I think that will be more clear and not so redundant.